src/react-components/home-root.js

```diff
@@ -255,38 +286,70 @@ class HomeRoot extends Component {
                   Pixel
                 </a>
               </div>
-              <div className={styles.container}>
-                <div className={styles.logo}>
-                  <img src={hubLogo} />
-                </div>
-                <div className={styles.title}>
-                  <FormattedMessage id="home.hero_title" />
-                </div>
-                {this.state.environments.length === 0 && (
-                  <div className="loader-wrap">
-                    <div className="loader">
-                      <div className="loader-center" />
+              <div className={styles.heroPanel}>
+                <div className={styles.container}>
+                  <div className={styles.logo}>
+                    <img src={hubLogo} />
+                  </div>
+                  <div className={styles.title}>
+                    <FormattedMessage id="home.hero_title" />
+                  </div>
+                  {isOculusBrowser && (
+                    <div className={styles.blurb}>
+                      <FormattedMessage id="home.hero_blurb" />
                     </div>
+                  )}
+                  {!isOculusBrowser &&
+                    this.state.environments.length === 0 && (
+                      <div className="loader-wrap">
+                        <div className="loader">
+                          <div className="loader-center" />
+                        </div>
+                      </div>
+                    )}
+                </div>
+                {isOculusBrowser ? (
+                  <div className={styles.ctaButtons}>
+                    <button
+                      className={classNames(styles.primaryButton, styles.ctaButton)}
+                      onClick={this.showCreateDialog}
+                    >
+                      <FormattedMessage id="home.create_a_room" />
+                    </button>
+                    <button className={classNames(styles.secondaryButton, styles.ctaButton)} onClick={this.launchTour}>
+                      <FormattedMessage id="home.take_a_tour" />
+                    </button>
+                  </div>
+                ) : (
+                  <div className={styles.create}>
+                    <HubCreatePanel
+                      initialEnvironment={this.props.initialEnvironment}
+                      environments={this.state.environments}
+                    />
                   </div>
                 )}
               </div>
-              <div className={styles.create}>
-                <HubCreatePanel
-                  initialEnvironment={this.props.initialEnvironment}
-                  environments={this.state.environments}
-                />
+              <div className={classNames(styles.heroPanel, styles.rightPanel)}>
+                {/*isOculusBrowser && (
```

Comment:
maybe put this into a software switch (just a local variable hard coded to false) vs a comment